src/SSHDebugPS/UI/ContainerPickerDialogWindow.xaml

```diff
@@ -30,12 +30,12 @@
         <Style x:Key="GlyphPathStyle"
                TargetType="{x:Type Path}">
             <Setter Property="Stroke"
-                            Value="#717171" /> <!-- GlyphBrushKey = Glyph = 717171-->
+                            Value="{StaticResource {x:Static SystemColors.GrayTextBrushKey}}" />
             <Style.Triggers>
                 <DataTrigger Binding="{Binding  RelativeSource={RelativeSource Mode=FindAncestor, AncestorType={x:Type ListBoxItem}}, Path=IsSelected}"
                              Value="True">
                     <Setter Property="Stroke"
-                            Value="#006CBE" /> <!-- GlyphMouseOverBrushKey = GlyphMouseOver = 006CBE-->
+                            Value="{StaticResource {x:Static SystemColors.HighlightBrushKey}}" />
```

Comment:
Did you mean to use HighlightBrushTextKey? Won't this look like blue glyph on blue row highlight?

Reply:
right, it doesn't meet the contrast requirements. And this also doesn't work well in high contrast scenarios.

Reply:
When using HighlightTextBrushKey, the contrast is not sufficient when High Contrast is off:
![image](https://user-images.githubusercontent.com/60245970/135688841-e4183415-23a2-47dc-bec7-6c3d5be748d4.png)

Blue Theme:
![image](https://user-images.githubusercontent.com/60245970/135688260-663f172e-4631-4d73-9909-57c8abd99a89.png)

High Contrast 1:
![image](https://user-images.githubusercontent.com/60245970/135688371-5a585c95-5ca4-4b3c-aa19-c69e578c2278.png)

High Contrast 2:
![image](https://user-images.githubusercontent.com/60245970/135688453-b7032d10-ed93-4a2b-b362-e60304701fac.png)

High Contrast Black:
![image](https://user-images.githubusercontent.com/60245970/135688504-05e8ee14-12e8-4ae3-8ae3-ed9673865542.png)

High Contrast White:
![image](https://user-images.githubusercontent.com/60245970/135688551-0f853723-02a6-4a93-95d6-5d43db2998b4.png)

Reply:
This is the blue glyph on blue row highlight mentioned above in high contrast off (this is the current implementation):
![image](https://user-images.githubusercontent.com/60245970/135689146-220e13f5-c9fe-4402-bbe6-0cf0cca71d08.png)